### PR TITLE
Create fahmedo -profile.json

### PR DESCRIPTION
**Pull request by Fahmedo**
Create a json file and add my profiles details

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/fahmedo -profile.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Faruq Ahmed Olaitan",
+  "bio": "Open Source Enthusiast | Frontend Engineer",
+  "links": [
+    {
+      "group": "Social primary",
+      "name": "X: Follow me",
+      "url": "https://twitter.com/da_horlie",
+      "icon": "FaXTwitter"
+    }
```
Comment (Sindhuinti) [inline data/fahmedo -profile.json:10]:
```suggestion
    },
```

File: data/fahmedo -profile.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    {
+      "group": "Social primary",
+      "name": "LinkedIn: Follow me",
+      "url": "https://www.linkedin.com/in/ahmed-faruq-6116a4232/",
+      "icon": "FaLinkedin"
+    }
```
Comment (Sindhuinti) [inline data/fahmedo -profile.json:16]:
```suggestion
    },
```

File: data/fahmedo -profile.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    {
+      "group": "Social primary",
+      "name": "Instagram: Follow me",
+      "url": "https://www.instagram.com/ola_kanbayi/",
+      "icon": "FaInstagram"
+    }
```
Comment (Sindhuinti) [inline data/fahmedo -profile.json:22]:
```suggestion
    },
```

**Review by Sindhuinti — CHANGES_REQUESTED** [review]
Please add comma as suggested

**Review by SaraJaoude — DISMISSED** [review]
Please make sure your file name matches your GitHub username exactly (it is case sensitive). Instead of `data/fahmedo -profile.json`   it should be `data/Fahmedo.json`

I would suggest you look at our QuickStart Guide  with JSON (especially section 7) in Docs: https://www.biodrop.io/docs/quickstart-json to help you update this Pull Request.

**Review by SaraJaoude — APPROVED** [review]
Thank you for making the changes, I can now merge this PR.

**Pull request merged**